backend/src/main/java/com/team4/backend/service/StudentService.java

```diff
@@ -98,4 +100,14 @@ public StudentService(StudentRepository studentRepository, PBKDF2Encoder pbkdf2E
                         Mono.error(new UserNotFoundException("Could not find student with id: " + studentId))
                 );
     }
+
+    public Mono<Long> updateStudentStateForAllStudentThatInterviewDateHasPassedWeb() {
+        return studentRepository.findAllByStudentStateAndInterviewsDateIsNotEmpty(StudentState.INTERNSHIP_NOT_FOUND)
+                .map(student -> {
+                    student.setStudentState(StudentState.WAITING_FOR_RESPONSE);
+                    log.info("STATE UPDATED : " + student.getFirstName() + ", " + student.getLastName());
```

Comment:
Log returning `INFO: STATE UPDATED : null, null` when running StudentServiceTest.java

Reply:
yeah i forgot to add firstName and lastName on the mockData i fixed it